scripts/ansible/aws/ec2-set-volume-size.yml

```diff
@@ -19,7 +19,8 @@
 
   - name: Modify the volume
     amazon.aws.ec2_vol:
-      profile: "{{ aws_profile }}"
```

Comment:
Are you removing it because Github does not allow having AWS profiles? And would it break people's workflow running the scripts on their machines? If yes, then can you make it default like "aws_profile | default(''"

Reply:
Once we have this running I would largely prefer to not have people running it locally. That being said, Oleg has expressed some frustration with having to authenticate using a browser since we switched over to the Google SSO for our AWS accounts. This method would allow us to avoid that I think (although might require a second CLI only account)